alts/worker/tasks.py

```diff
@@ -95,7 +95,9 @@ def is_success(stage_data_: dict):
                      if task_params.get('repositories') else [],
                      'dist_arch': task_params.get('dist_arch')
                      if task_params.get('dist_arch') else 'x86_64'}
-
+    log_name = f"hui-{task_id}-{arch}.{ts}.log"
```

Comment:
Are these logs to be stored on the host? At some point they will result a "no space left"

Reply:
that was a debug lines to help Maxim with his task and I added them by accident, I will remove it